corehq/apps/data_interfaces/views.py

```diff
@@ -450,3 +456,84 @@ def post(self, request, *args, **kwargs):
                 return HttpResponseRedirect(self.page_url)
             return self.get(request, *args, **kwargs)
         return self.paginate_crud_response
+
+
+class XFormManagementView(DataInterfaceSection):
+    urlname = 'xform_management'
+    page_title = ugettext_noop('Form Management')
+
+    def get_form_ids_or_query_string(self, request):
+        if 'select_all' in self.request.POST:
+            # Altough evaluating form_ids and sending to task would be cleaner,
+            # heavier calls should be in in an async task instead
+            import urllib
+            form_query_string = urllib.unquote(self.request.POST.get('select_all'))
```

Comment:
Should you not be returning the entire query string and not just the 'select_all' parameter?